sql/src/main/java/org/apache/druid/sql/calcite/expression/builtin/LPadOperatorConversion.java

```diff
@@ -0,0 +1,85 @@
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.apache.druid.sql.calcite.expression.builtin;
+
+import com.google.common.collect.ImmutableList;
+import org.apache.calcite.rex.RexNode;
+import org.apache.calcite.sql.SqlFunction;
+import org.apache.calcite.sql.SqlFunctionCategory;
+import org.apache.calcite.sql.SqlOperator;
+import org.apache.calcite.sql.type.SqlTypeFamily;
+import org.apache.calcite.sql.type.SqlTypeName;
+import org.apache.druid.sql.calcite.expression.DruidExpression;
+import org.apache.druid.sql.calcite.expression.OperatorConversions;
+import org.apache.druid.sql.calcite.expression.SqlOperatorConversion;
+import org.apache.druid.sql.calcite.planner.PlannerContext;
+import org.apache.druid.sql.calcite.table.RowSignature;
+
+public class LPadOperatorConversion implements SqlOperatorConversion
+{
+  private static final SqlFunction SQL_FUNCTION = OperatorConversions
+      .operatorBuilder("LPAD")
+      .operandTypes(SqlTypeFamily.CHARACTER, SqlTypeFamily.INTEGER, SqlTypeFamily.CHARACTER)
+      .returnType(SqlTypeName.VARCHAR)
+      .functionCategory(SqlFunctionCategory.STRING)
+      .requiredOperands(2)
+      .build();
+
+  @Override
+  public SqlOperator calciteOperator()
+  {
+    return SQL_FUNCTION;
+  }
+
+  @Override
+  public DruidExpression toDruidExpression(
+      final PlannerContext plannerContext,
+      final RowSignature rowSignature,
+      final RexNode rexNode
+  )
+  {
+    return OperatorConversions.convertCall(
+        plannerContext,
+        rowSignature,
+        rexNode,
+        druidExpressions -> {
+          if (druidExpressions.size() > 2) {
+            return DruidExpression.fromFunctionCall(
+                "lpad",
+                ImmutableList.of(
+                    druidExpressions.get(0),
+                    druidExpressions.get(1),
+                    druidExpressions.get(2)
+                )
+            );
+          } else {
+            return DruidExpression.fromFunctionCall(
+                "lpad",
+                ImmutableList.of(
+                    druidExpressions.get(0),
+                    druidExpressions.get(1),
+                    DruidExpression.fromExpression(DruidExpression.stringLiteral(" "))
+                )
+            );
+          }
+        }
+    );
+  }
+}
```

Comment:
Nit - please include a newline at the end of the file

Reply:
Hi, @justinborromeo , thanks for review. It seems that this file is ended as `0x7d 0x0a` which are `}\n`. From [raw file](https://raw.githubusercontent.com/apache/incubator-druid/7d3474ac9cea4e444350034a615f83df6e5d1dc9/sql/src/main/java/org/apache/druid/sql/calcite/expression/builtin/LPadOperatorConversion.java) it looks having a blank ending line. Most other java files seems the same. I  an not sure whether I understood correctly..Thank you